modules/puppet/templates/govuk_puppet

```diff
@@ -191,10 +191,12 @@ if [[ " ${puppet_args} " =~ "--enable " ]]; then
   remove_lock
 fi
 
-locked=$(has_locks)
-if [ "$locked" == 1 ];then
-  list_locks
-  exit 1
+if [[ ! " ${puppet_args} " =~ "--noop " ]]; then
```

Comment:
Why are there spaces around the argument and the `puppet_args` variable?

Reply:
I just copied the format of the `--enable` and others, (for example: `if [[ " ${puppet_args} " =~ "--enable " ]]; then`).

`=~` is a regular expression based match (I think), so I can see why you want a space at the end, as there is a space at the end of `--noop ` to guard against matching things like `--noop-foo` (made up example). I don't know why there is a space at the start, I assume it would work without it.

Reply:
That makes sense, thanks!